.github/workflows/pr-reminder.yml

```
@@ -2,7 +2,7 @@ name: PR Reminder
 
 on:
   schedule:
-    - cron: "0 0,5,8 * * *" # 아침 9시, 오후 2시, 오후 5시에 실행 (UTC 기준으로 설정해서 한국 시간에 맞춤)
+    - cron: "47 23,4,7,8, 10 * * *" # 아침 8시 47분, 오후 2시 47분, 오후 4시 47분, 오후 5시 47분, 오후 7시 47분 에 실행 (UTC 기준으로 설정해서 한국 시간에 맞춤)
```

actionlint (1.7.8)

invalid CRON format "47 23,4,7,8, 10 * * *" in schedule event: expected exactly 5 fields, found 6: [47 23,4,7,8, 10 * * *]

(events)